src/mixins/resource-lookup-mixin.ts

```diff
@@ -8,12 +8,13 @@ import { FilingCodes } from '@/enums'
  */
 @Component({
   computed: {
-    ...mapState(['configObject'])
+    ...mapState(['configObject', 'isBComp'])
   }
 })
 
 export default class ResourceLookupMixin extends Vue {
     readonly configObject!: any
+    readonly isBComp: boolean
```

Comment:
Add non-null assertion operator.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
kk, will do!

Reply:
Updated in next commit

Reply:
I think that operator is standard practice for store properties. I dunno what happens if the property is actually supposed to be null...?

Reply:
Thats good question. I suppose in that scenario a non-null assertion wouldn't be wanted?